bundler/spec/commands/update_spec.rb

```diff
@@ -613,6 +613,12 @@
       expect(err).to match(/You are trying to install in deployment mode after changing.your Gemfile/m).
         and match(/freeze \nby running `bundle config unset deployment`./m)
     end
+
+    it "should not suggest any command to unfreeze bundler is freezing is set through ENV" do
```

Comment:
Should be `...if freezing...`?

Reply:
Maybe this better?

```suggestion
    it "should not suggest any command to unfreeze bundler is frozen is set through ENV" do
```

Reply:
There is `is` twice, it seems to me one of the `is` should be actually `if`. But maybe I miss something.

Reply:
Oh, sorry, no no, I misread it! Will fix, thanks!